### DISCUSSION TITLE
Complete listing of @triggered commands

### DISCUSSION THREAD

**Discussion by TryerGit** [James-Yu/LaTeX-Workshop#3636] [Q&A] [answered]
Hello,

At present, there seem to only be a proper subset of all `@` triggered commands available on the home page [here](https://github.com/James-Yu/LaTeX-Workshop/wiki/Intellisense#-suggestions).

Could you please point me to the full list? For e.g., I was typing `\setminus` and I got a hint popup stating that `@\` is its shortcut. However, I was not able to find it on the homepage listing above.

Thank you.

**Comment by jlelong** [comment, marked as answer]
See https://github.com/James-Yu/LaTeX-Workshop/blob/5464ebc321a3107a0d7e0f7ed7130b357e0d0c6a/data/at-suggestions.json
